CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 The changelog for [Kommunicate-iOS-SDK](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK). Also see the [releases](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK/releases) on Github.
 
 ## [Unreleased]
+- [TD-1700] Added code to update group metadata while creating a conversation
```

Comment:
We don't need to mention `added code`. It should be `Added suppot to update group metadata while creating a conversation`

Reply:
This one is pending

Reply:
Alright